### ISSUE TITLE
[TW-1059] task wait:eom  -  bulk or not, and why/when

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1094]
_Thomas Sattler on 2010-12-23T04:41:24Z says:_ 

 I scanned my tasks, what to do this year and what to postpone. As I understood the tutorial, I can put a wait on several tasks at once, but I have to confirm the individual task (yes, I know about 'All'). But I don't understand when confirmation is triggered: Postponing 'only' two tasks -> no confirmation, postponing three tasks -> confirmation:

$ task 15,16 wait:eom
Modified 2 tasks.

$ task 3,4,20 wait:eom

Task 3 "learn task"
*  wait will be set to '31.12.2010'.
*  status will be changed from 'pending' to 'waiting'.
Proceed with change? (Yes/no/All/quit)

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2010-12-23T04:41:24Z
Modified: 2014-02-09T02:16:21Z
```

**Comment by taskwarrior** [comment]
_Federico Hernandez on 2010-12-23T05:08:04Z says:_ 

 There is a configuration variable called bulk:

<pre>
$ task show bulk

Config variable Value
bulk            2
</pre>

From the man page taskrc:

bulk=2 Is a number, defaulting to 2. 
When more than this number of tasks are modified in a single command, confirmation will be required, unless the confirmation variable is "no". This is useful for preventing large-scale unintended changes.
